app/templates/events/volunteerDetails.html

```diff
@@ -19,23 +19,29 @@
+  
+    <div class="{{volunteerType}}Select col-12 col-md-6 volunteerInfoEntries print-column" data-user="{{participant.user.username}}" nowrap>
+      <h4 class="nameSelect" nowrap><b>{{ participant.user.firstName }} {{ participant.user.lastName }}</b></h4>
       <ul>
-        <div class="barcodeSelect bnumber-barcode" nowrap>{{participant.bnumber[1:]}}</div>
-        <div class="phoneSelect" nowrap>{{ participant.phoneNumber if participant.phoneNumber else "No phone number"}}</div>
-        <div class="emailSelect" nowrap>{{ participant.email }}</div>
-        <div class="dietRestrictionSelect">Dietary Restrictions: {{ participant.dietRestriction if participant.dietRestriction else "None Specified" }}</div>
+        <div class="barcodeSelect bnumber-barcode" nowrap>{{participant.user.bnumber[1:]}}</div>
+        <div class="phoneSelect" nowrap>{{ participant.user.phoneNumber if participant.user.phoneNumber else "No phone number"}}</div>
+        <div class="emailSelect" nowrap>{{ participant.user.email }}</div>
+        <div class="dietRestrictionSelect">Dietary Restrictions: {{ participant.user.dietRestriction if participant.user.dietRestriction else "None Specified" }}</div>
+        <div class="emergencyContactSelect">Emergency Contact: {{participant.emergencycontact.name if participant.emergencycontact}} 
+          {{participant.emergencycontact.cellPhone or participant.emergencycontact.homePhone or participant.emergencycontact.workPhone if participant.emergencycontact}} 
+          {{"None Specified" if not participant.emergencycontact}}</div>
       </ul>
     </div>
 {% endmacro %}
 
 {% macro createTable(participant, volunteerType) %}
-    <tr class="{{volunteerType}}Select volunteerInfoEntries" data-user="{{participant.username}}">
-      <td class="nameSelect" nowrap><b>{{ participant.firstName }} {{ participant.lastName }}</b></td>
-      <td class="phoneSelect" nowrap>{{ participant.phoneNumber if participant.phoneNumber else "" }}</td>
-      <td class="emailSelect" nowrap>{{ participant.email }}</td>
-      <td class="dietRestrictionSelect">{{ participant.dietRestriction if participant.dietRestriction else ''}}</td>
+    <tr class="{{volunteerType}}Select volunteerInfoEntries" data-user="{{participant.user.username}}">
+      <td class="nameSelect" nowrap><b>{{ participant.user.firstName }} {{ participant.user.lastName }}</b></td>
+      <td class="phoneSelect" nowrap>{{ participant.user.phoneNumber if participant.user.phoneNumber else "" }}</td>
+      <td class="emailSelect" nowrap>{{ participant.user.email }}</td>
+      <td class="dietRestrictionSelect">{{ participant.user.dietRestriction if participant.user.dietRestriction else ''}}</td>
+      <td class="emergencyContactSelect">{{participant.emergencycontact.name if participant.emergencycontact}} 
+        {{participant.emergencycontact.cellPhone or participant.emergencycontact.homePhone or participant.emergencycontact.workPhone if participant.emergencycontact}} </td>
```

Comment:
You might need to consider handling the case in which there is no emergency contact.

Reply:
The case for no emergency contact is handled by leaving the table cell blank.